.travis.yml

```diff
@@ -1,20 +1,35 @@
 ---
 language: node_js
+node_js:
+    - "0.12"
 
 sudo: false
 
 cache:
-  directories:
-    - node_modules
+    directories:
+        - node_modules
+
+env:
+    - EMBER_TRY_SCENARIO=default
+    - EMBER_TRY_SCENARIO=ember-release
+    - EMBER_TRY_SCENARIO=ember-beta
+    - EMBER_TRY_SCENARIO=ember-canary
+
+matrix:
+    fast_finish: true
+    allow_failures:
+        - env: EMBER_TRY_SCENARIO=ember-beta
+        - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-  - "npm config set spin false"
-  - "npm install -g npm@^2"
+    - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
```

Comment:
I went ahead and removed it, but I don't really understand the direct problem since Travis was building fine with that line in there...